themes/digital.gov/layouts/partials/core/featured-collection.html

```diff
@@ -0,0 +1,103 @@
+{{/* Gets the full array of items passed into the partial */}}
+{{- $list := .list -}}
+
+{{/* If there are any items in the array at all... */}}
+{{- if $list -}}
+  <div class="featured-collection">
+    {{/* Checks if there a heading is set in the partial */}}
+    {{- if .heading -}}
+      <h2>{{- .heading -}}</h2>
+    {{- end -}}
+
+
+    <ul>
+      {{- range $list -}}
+
+        {{/* If an external link */}}
+        {{- if or .Params.source_url .Params.source -}}
+
+          {{/* If there is a defined source for this page */}}
+          {{- if .Params.source -}}
+
+            {{/* Get the source data */}}
+            {{- $source := .Site.GetPage (print "source_" .Params.source ) -}}
+
+            {{/* If there is a logo defined in the source */}}
+            {{- if $source.Params.logo -}}
+
+              {{/* get the path for the icon — e.g. 18f-icon.png */}}
+              {{- $src := (printf "/logos/%s%s" $source.Params.logo "-icon.png") -}}
+              {{/* Pass these vars into the collection template
+                (see bottom of collection.html partial)
+              */}}
+              {{- template "collection-list-item" dict "item" . "href" (print .Params.source_url "?dg") "src" $src -}}
+
+            {{- end -}}
+
+            {{/* No source? let's attempt to get the favicon for the URL */}}
+          {{- else -}}
+
+            {{/* get the Favicon via Google Favicon service */}}
+            {{- $src := (print "https://www.google.com/s2/favicons?domain=" .Params.source_url) -}}
+            {{/* Pass these vars into the collection template
+              (see bottom of collection.html partial)
+            */}}
+            {{- template "collection-list-item" dict "item" . "href" (print .Params.source_url "?dg") "src" $src -}}
+
+          {{- end -}}
+
+          {{/* If an internal link... */}}
+        {{- else -}}
+
+          {{/* Use the digital.gov logo */}}
+          {{/* Pass these vars into the collection template
+            (see bottom of collection.html partial)
+          */}}
+          {{- template "collection-list-item" dict "item" . "href" (print .Permalink "?dg") "src" (print "/img/digit-50.png") -}}
+
+        {{- end -}}
+      {{- end -}}
+    </ul>
+  </div>
+{{- end -}}
+
+{{- define "collection-list-item" -}}
```

Comment:
What's the difference between `collection-list-item` and `/layouts/partials/core/collection-list-item.html`?

Reply:
Different path's for internal templates versus external partials.